src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.data.person;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.tag.UniqueTagList;
+
+public class Tagging {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.